workflow-templates/publish-mvn-package-in-grp.yml

```diff
@@ -0,0 +1,39 @@
+name: Publish Java Package mvn in GPR
```

Comment:
Nesse aqui só mudaria o nome do aquivo mesmo para `publish-java-package-mvn-in-grp.yml`. Para ficar de acordo com workflow name.